dashboard/test/factories.rb

```diff
@@ -516,6 +516,13 @@
     conditionals_d5_count 4
   end
 
+  factory :user_profile do
+    user { create :teacher }
```

Comment:
Minor: It's more flexible to write this as `association :user, factory: :teacher` in case we ever want to build instead of create (and not commit to the db). Not a big deal though if we're not using that.